index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>End of the World Event</title>
+    <link rel="icon" type="image/x-icon" href="ressources/event-image/EoW-red-circle.png">
+
+  </head>
+  <body>
+    <header class="header">
+      <section class="black-banner-container">
+        <div class="black-banner-items">
+          <a href="https://facebook.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-facebook"></a>
+          <a href="https://twitter.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-twitter"></a>
+          <a href="#English" target="_blank" rel="noopener">English</a>
+          <a href="#MyPage" target="_blank" rel="noopener">My Page</a>
+          <a href="#Logout" target="_blank" rel="noopener">Logout</a>
+        </div>
+      </section>
+      <section class="white-banner">
+        <div class="logo-container">
+          <a href="/" class="logo-container-2">
+            <img src="ressources/event-image/EoW-Logo.png" alt="A red logo for the End of the World Event" class="logo">
+          </a>
+        </div>
+        <div class="menu-container">
+          <div id="overlay-screen" class="overlay">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+            <nav class="overlay-content menu">
+              <a href="./about.html">About</a>
+              <a href="#Program">Program</a>
+              <a href="#Jobs">Jobs</a>
+              <a href="#Sponsor">Sponsor</a>
+              <a href="#News">News</a>
+            </nav>
+          </div>
+          <span class="hamburger" onclick="openNav()">&#9776;</span>
+        </div>
+        <nav class="nav-menu">
+          <a href="./about.html">About</a>
+          <a href="#Program">Program</a>
+          <a href="#Jobs">Jobs</a>
+          <a href="#Sponsor">Sponsor</a>
+          <a href="#News">News</a>
+        </nav>
+        <div class="button-container">
+          <button class="header-button" type="button" name="button">CC Campaing</button>
+        </div>
+      </section>
```

Comment:
- [ ] In about page the hamburger button does not work so there is no way for the user to go back to the main page using UI, please update it

Reply:
- [ ] The hamburger button should be more in the left and please adjust the color based on the design.

Yours:
![Screenshot from 2021-11-16 22-40-46](https://user-images.githubusercontent.com/62705947/142070405-0f9e10dc-ec36-44f5-8fcc-5da505195cb5.png)
GIven design:
![Screenshot from 2021-11-16 22-40-49](https://user-images.githubusercontent.com/62705947/142070442-16ba2eb3-6022-49b5-88bc-38451c592ce8.png)

Reply:
- [ ] In the desktop version please move the nav-menu items more in to the right in order to match perfectly the given design

Reply:
Thanks for pointing this out, the necessary changes have been made